scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,60 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    r"""
+    Khatri-rao product
+    A column-wise Kronecker product of two matrices
+
+    Parameters
+    ----------
+    a:  (n, k) arraylike
+        Input array
+    b:  (m, k) arraylike
+        Input array
+
+    Returns
+    -------
+    c:  (n*m, k) ndarray
```

Comment:
This won't render well, because `*` is a reST special character. I would move the shape specification into the description (also for `a` and `b` above).